deploy/backplane/20-cssre.SubjectPermission.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: backplane-cssre
+  namespace: openshift-rbac-permissions
+spec:
+  clusterPermissions:
+  - backplane-cssre-read-all
+  - backplane-cssre-admin-cluster
+  permissions:
+  - allowFirst: true
+    clusterRoleName: backplane-cssre-admin-project
+    namespacesAllowedRegex: (^redhat-rhmi-*)
```

Comment:
Does not match what CSSRE has today.  They're administering everything but the platform in the current setup.  If we change it, we need to have signoff from cssre.

Reply:
Thanks, I will change it to what they have today from [layered-ccsre-subjectpermission](https://github.com/openshift/managed-cluster-config/blob/master/deploy/layered-sre-authorization/02-layered-cs-sre-admin.SubjectPermission.yaml)